CorinneTest/EmptyClass.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+namespace CorinneTest
+{
+    public class EmptyClass
```

Comment:
Why do we have an empty class?  Why is it also called "EmptyClass"?

Reply:
thought that I deleted this - now deleted.